### ISSUE TITLE
firebase config: use clean URLs w/o trailing slashes

### ISSUE THREAD

**Issue by chalin** [flutter/website#1126]
_From @chalin on July 25, 2018 15:43_

The current flutter.io site seems to enforce trailing slashes everywhere. Was there an original reason for not opting for clean URLs clean URLs w/o trailing slashes? Firebase supports URL cleanup (via redirects) for these options:

```json
{
  "hosting": {
    "...": "...",
    "cleanUrls": true,
    "trailingSlash": false,
    "...": "..."
  }
}
```

cc @Sfshaza @filiph 

_Copied from original issue: flutter/flutter#19769_

**Comment by chalin** [comment]
_From @Sfshaza on August 1, 2018 15:25_

If there was a reason, I don't remember. (It seems like it was needed on the most recent version of the *.dartlang sites). But if it's not needed, I'm happy with that.

**Comment by chalin** [comment]
Closed by #1225
